src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -424,6 +440,25 @@ private CallbackResponse setApplicantResponseDeadline(CallbackParams callbackPar
             .applicant1ResponseDeadline(getApplicant1ResponseDeadline(responseDate, allocatedTrack))
             .businessProcess(BusinessProcess.ready(DEFENDANT_RESPONSE_SPEC));
 
+        updatedData.respondent1DetailsForClaimDetailsTab(updatedRespondent1);
+
+        if (ofNullable(caseData.getRespondent2()).isPresent()
```

Comment:
this use of ofNullable is the same than caseData.respondent2 != null && caseData.respondent1Copy != null, using Optional does not add worth